src/main/java/salesforce/ui/pages/LoginPage.java

```diff
@@ -9,20 +8,12 @@
     @FindBy(name = "username")
     private WebElement usernameTxtBox;
 
-    @FindBy(name = "password")
+    @FindBy(name = "pw")
```

Comment:
PAO is this locator right? I think the locator name is password

Reply:
yeah.. I don't know why .. it didn't work with password